source/sync/get-started.txt

```diff
@@ -69,17 +73,17 @@ check out the SDK docs:
 
 .. note::
 
-   The Realm Web SDK does not currently support Realm Database or Atlas Device Sync.
-   However, you can query the same data in an Atlas cluster using 
+   The Web SDK does not currently support Realm Database or Atlas Device Sync.
+   However, you can query the same data in an Atlas cluster using
    :ref:`MongoDB Data Access <web-mongodb-data-access>`.
 
 Define Data Model
 ~~~~~~~~~~~~~~~~~
 
-A synced realm uses object models that you define to determine the type of
-objects in the realm. Additionally, Sync requires a server-side document schema
+A synced database uses object models that you define to determine the type of
+objects in the database. Additionally, Sync requires a server-side document schema
 for each synced object type so that App Services can translate and apply changes
```

Comment:
We may want to remove "App Services" from here since it's now possible to use Device Sync without going through App Services.
```suggestion
for each synced object type so the server can translate and apply changes
```